lib/rexml/source.rb

```diff
@@ -87,7 +87,7 @@ def initialize(arg, encoding=nil)
         detect_encoding
       end
       @line = 0
-      @term_encord = {}
+      @encoded_term = {}
```

Comment:
Minor: How about making it a plural?

```suggestion
      @encoded_terms = {}
```